go-controller/pkg/ovn/ipallocator/allocator/bitmap_test.go

```diff
@@ -155,3 +155,173 @@ func TestContiguousAllocation(t *testing.T) {
 		t.Errorf("unexpected success")
 	}
 }
+
+func TestRoundRobinAllocation(t *testing.T) {
+	max := 10
+	m := NewRoundRobinAllocationMap(max, "test")
+
+	for i := 0; i < max; i++ {
+		next, ok, _ := m.AllocateNext()
+		if !ok {
+			t.Fatalf("unexpected error")
+		}
+		if next != i {
+			t.Fatalf("expect next to %d, but got %d", i, next)
+		}
+	}
+
+	if _, ok, _ := m.AllocateNext(); ok {
+		t.Errorf("unexpected success")
```

Comment:
Why are some of the failures here fatal vs. some errors?

Reply:
@abhat no clue, it's literally a C&P of the existing tests with s/NewAllocator/NewRoundRobinAllocator. Can change if you like...

Reply:
[done]